site2/docs/functions-package.md

```diff
@@ -0,0 +1,524 @@
+---
+id: functions-package
+title: Package Pulsar Functions
+sidebar_label: "How-to: Package"
+---
+
+This section provides step-by-step instructions to package Pulsar functions in Java, Python, and Go. 
+
+> **Tip**
+>
+> - Packaging a window function in Java is the same as [packaging a function in Java](#java) as below. 
+>
+> - Currently, the window function is not available in Python and Go.
+
+# Prerequisite
+
+Before running a Pulsar function, you need to start up Pulsar.
+
+## Run Pulsar cluster in Docker
+
+This example uses Docker to run a standalone Pulsar.
+
+```bash
+docker run -it \
+    -p 6650:6650 \
+    -p 8080:8080 \
+    -v $PWD/data:/pulsar/data \
+    apachepulsar/pulsar:latest \
+    bin/pulsar standalone
+```
+
+> **Tip**
+>
+> - `$PWD/data` is the local directory. `-v` maps the `/pulsar/data` directory in the Docker image to the local `$PWD/data` directory.
+>
+> - To check whether the image starts up or not, use the command `docker ps`.
+
+## Run Pulsar cluster in k8s
+
+For how to deploy Pulsar cluster in the k8s environment, please refer to [here](https://pulsar.apache.org/docs/en/helm-overview/).
+
+
+# Java 
+
+This example demonstrates how to package a function in Java.
+
+> **Note**
+>
+> This example assumes that you have [run a standalone Pulsar in Docker](#Run Pulsar cluster in Docker) successfully.
```

Comment:
the link is not correct.

Reply:
1. run a standalone Pulsar in Docker or run Pulsar cluster in Docker?
2. How about run Pulsar in K8s? Do the following steps work for K8s?